libpod/image/image.go

```diff
@@ -174,12 +176,17 @@ func (i *Image) reloadImage() error {
 
 // stringSha256 strips sha256 from user input
 func stripSha256(name string) string {
-	if strings.HasPrefix(name, "sha256:") && len(name) > 7 {
+	if strings.HasPrefix(name, fmt.Sprintf("%s:", ImageDigestType)) && len(name) > 7 {
 		return name[7:]
 	}
 	return name
 }
 
+// PrependImageDigestTypeToID
```

Comment:
Better comment?

Reply:
Also, wait, isn't the image's digest a SHA256, while the ID is just a 256-bit random integer?

Reply:
are we talking about semantics here?